ph1_webapp/index.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="ja">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>webapp</title>
+  <link rel="stylesheet" href="./styles/reset.css">
+  <link rel="stylesheet" href="./styles/style.css">
+</head>
+<body id="body">
+  <header class="header">
+    <div class="header-left">
+      <img src="./img/logo.svg" alt="logo">
+      <p class="header-week-cnt" id="thisWeek"></p>
+    </div>
+    <button class="post" id="post">記録・投稿</button>
+  </header>
+  <main class="main">
+    <div class="left">

```

Comment:
header-leftっていうクラス名上で使ってたからルール的に
```suggestion
    <div class="main-left">

```

Reply:
main-left、main-rightに修正しました！